pkg/proc/bininfo.go

```diff
@@ -1575,6 +1589,10 @@ func (bi *BinaryInfo) setGStructOffsetElf(image *Image, exe *elf.File, wg *sync.
 
 		bi.gStructOffset = tlsg.Value + uint64(bi.Arch.PtrSize()*2) + ((tls.Vaddr - uint64(bi.Arch.PtrSize()*2)) & (tls.Align - 1))
 
+	case elf.EM_PPC64:
```

Comment:
How do the tests pass without gStructOffset being set?

Reply:
The address is stored directly in a register, so we never reference the `bi.gStructOffset` symbol (see `PPC64LERegisters.GAddr`.

Reply:
Can we remove this?

Reply:
I think we need it if we remove the case it fails with a `panic: architecture not supported`